workflow/landing-page/index.js

```diff
@@ -9,3 +9,121 @@ document.querySelector('.close-button').addEventListener('click', function () {
 document.querySelector('.menu').addEventListener('click', function () {
   document.querySelector('.navigation-menu').classList.toggle('opened');
 });
+
+const APIKEY = '4qYrEoyFHTFZXZNSPArlJHKDjcaSUbD2';
+
+let currentOffset = 0;
+let content;
+
+const nextGif = document.querySelector('#next-page');
+const previousGif = document.querySelector('#previous-page');
```

Comment:
because of the names not clear what this elements do.
Better to name smth like this: `nextPageBtn`, `previousPageBtn`